userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -39,7 +39,7 @@ void application::check_for_ignored_events()
 			continue;
 		}
 
-		if(!sinsp::simple_consumer_consider_evtnum(evtnum))
+		if(!simple_consumer_consider(etable[evtnum].flags, false))
```

Comment:
`simple_consumer_consider` is now a Falco only concept.

Reply:
Uhm here probably is not so simple, we need to check if this event is considered or not by Falco, so we need to check our set `ppm_sc_of_interest`. The problem is that Falco has no concepts of `event` but it only has concept of `ppm_sc` code, maybe we can provide a method in sinps to translate it :thinking:

Reply:
I agree with you; but we are already storing set used by Falco; i think enriching this check by also checking that the `ppm_sc` linked to each evtnum are present in Falco ppm_sc set should work.

Reply:
Waiting on https://github.com/falcosecurity/libs/pull/609 for this